<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Expenses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Text Search</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLineEdit" name="filter_text"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Appro&amp;ved Status</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="filter_approved_status"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="filter_from_box">
            <property name="title">
             <string>From</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QDateEdit" name="filter_from">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="filter_to_box">
            <property name="title">
             <string>To</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QDateEdit" name="filter_to">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ExpenseView" name="expense_view"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="users_view">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashDotDotLine</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
    <addaction name="action_Print_Report"/>
   </widget>
   <widget class="QMenu" name="menu_Expenses">
    <property name="title">
     <string>&amp;Expenses</string>
    </property>
    <addaction name="action_create_expense"/>
    <addaction name="action_Delete_Expense"/>
    <addaction name="action_Approve_Expenses"/>
   </widget>
   <widget class="QMenu" name="menu_Users">
    <property name="title">
     <string>&amp;Users</string>
    </property>
    <addaction name="action_create_new_user"/>
    <addaction name="actionUpdate_Model"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Expenses"/>
   <addaction name="menu_Users"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_create_expense">
   <property name="text">
    <string>&amp;Add Expense...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+E</string>
   </property>
  </action>
  <action name="action_Delete_Expense">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Expense</string>
   </property>
  </action>
  <action name="action_create_new_user">
   <property name="text">
    <string>&amp;Create New User...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+U</string>
   </property>
  </action>
  <action name="actionUpdate_Model">
   <property name="text">
    <string>&amp;Update Model</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Approve_Expenses">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Approve &amp;Expenses</string>
   </property>
  </action>
  <action name="action_Print_Report">
   <property name="text">
    <string>&amp;Print Report...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpenseView</class>
   <extends>QTableView</extends>
   <header>ExpenseView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
